react-electron-poc/src/renderer/components/panels/Erb/Erb.css

```diff
@@ -2,7 +2,7 @@
  * @NOTE: Prepend a `~` to css file paths that are in your node_modules
  *        See https://github.com/webpack-contrib/sass-loader#imports
  */
-.wrapper {
+.erb-wrapper {
```

Comment:
I hadn't seen erb before, but when I googled erb it looked really cool and helpful. Nice work!